dashboard/app/models/script.rb

```diff
@@ -1367,7 +1368,8 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       background: background,
       is_migrated: is_migrated?,
       scriptPath: script_path(self),
-      showCalendar: show_calendar
+      showCalendar: is_migrated ? show_calendar : false, #prevent calendar from showing for non-migrated scripts for now
```

Comment:
I think it might be good to add a test to make sure this summarizes correctly for the different cases. Let me know if you want an example of a test for this.

Reply:
Actually it looks like there was already a test for this and its failing in the drone run. So updating that test/adding ones similar to it for other cases would be good